tests/random-data-set.bats

```diff
@@ -100,3 +100,32 @@
   echo "output = ${output}"
   [ "$status" -eq 0 ]
 }
+
+
+@test "TSNE: Check that load_db_scxa_tsne.sh is in the path" {
+  run which load_db_scxa_tsne.sh
+  echo "output = ${output}"
+  [ "$status" -eq 0 ]
+}
+
+@test "TSNE: Create table" {
+  run psql $dbConnection < $testsDir/tsne/01-optional-create-table.sql
+  echo "output = ${output}"
+  [ "$status" -eq 0 ]
+}
+
+@test "TSNE: Load data" {
+  export EXP_ID=TEST-EXP1
+  run load_db_scxa_tsne.sh
+  echo "output = ${output}"
+  [ "$status" -eq 0 ]
+}
+
+@test "TSNE: Check number of loaded rows" {
+  # Get third line with count of total entries in the database after our load
+  count=$(echo "SELECT COUNT(*) FROM scxa_tsne" | psql $dbConnection | awk 'NR==3')
+  # TODO improve, highly dependent on test files we have, but in a hurry for now.
+  run [ $count -eq 250 ]
```

Comment:
Any particular reason why are we expecting counts 250 ?

Reply:
this is based on the test files I'm using really... not ideal, but I wanted a quick test.